validate/__main__.py

```diff
@@ -9,8 +9,7 @@ def parse_args():
     parser.add_argument('path', help='one or more paths of files to validate', type=str, nargs='+')
     parser.add_argument('-t', '--type', help='input data type',
         choices=['file-input', 'file-bam', 'file-vcf', 'file-fasta', 'file-fastq',
-        'file-bed', 'file-py', 'directory-rw', 'directory-r'],
-        default='file-input')
+        'file-bed', 'file-py', 'directory-rw', 'directory-r'])
```

Comment:
If most of these options are no longer used, we may as well update this to keep only the directory types and a single option for files: `choices=['file', 'directory-r', 'directory-rw']`

Reply:
I'll try that out and see if it would break if I specified a file-type. I wasn't sure if we wanted to just deprecate it and then remove it in a new major version.

Reply:
okay so the file-type choices from before won't work, as expected, but I think it's fine. people will just have to delete the map for the file type or change it to 'file' instead once they use the newer docker images. I checked and it definitely works with or without specifying "file" if they do change it or remove it.